rally_ovs/plugins/ovs/deployment/engines/ovs/install.sh

```diff
@@ -71,6 +71,15 @@ function install_ovs {
     sudo chown $OVS_USER /usr/local/var/log/openvswitch
 }
 
+function path_prepend {
```

Comment:
I guess you can simply use 
`export PATH="/usr/sbin:/usr/local/sbin:$PATH"`
here, duplicate path is not harm ;)

`export` is for if other script is called in turn, the PATH still contains "/usr/sbin:/usr/local/sbin".

Reply:
Perhaps I did over-engineer it :-) I sure can switch to just setting the `PATH`. If we do that, I think we need to append the might-be-missing entries so as not to override system's default configuration, in case it matters.

That said, I'm still not convinced that adjusting the `PATH` in our scripts is the right solution. It feels like it should be done outside of the scripts but I wasn't able to come up with a way to do it.

Reply:
Sure, sorry if I wasn't clear. By might-be-missing-entries I meant the ones we will be appending to `PATH` because we expect they might not be there yet. This is `/usr/local/sbin` and `/usr/sbin` in this case.

Why I think we need to append instead of prepend? User or sys admin can set `PATH` in `~/.bashrc` or `/etc/bashrc`, for example to point it to a custom location for OVS/OVN binaries. If we prepend to `PATH` we will enforce favouring the binaries that live in `*/sbin` where binaries from system packages or locally installed ones live.

Reply:
I haven't found any viable alternative to setting the `PATH` in the scripts.

The alternative options that I've considered are:
1. Setting PATH in `~/.ssh/environment` for `rally` user, requires `sshd` to have `PermitUserEnvironment` enabled which is not the default.
2. Setting PATH in `~/.bashrc` or sourcing `/etc/profile` from `~/.bashrc`.

Neither sounds simpler or cleaner to me than setting the `PATH` in the script itself so I will just do that.